src/index.js

```diff
@@ -0,0 +1,131 @@
+
+    this.renderMenu();
+
+    this.content = document.createElement('div');
+    this.content.className = 'content';
+
+    this.container.appendChild(this.content);
+
+    this.pageContent.appendChild(this.container);
+    this.wrapper.appendChild(this.pageContent);
+
+    this.app.appendChild(this.blurred);
+    this.app.appendChild(this.wrapper);
+  }
+
+  renderHeader() {
+    this.head = document.createElement('div');
+    this.head.className = 'head';
+    this.headBody = document.createElement('div');
+    this.headBody.className = 'head-body';
+    this.headBodyTitle = document.createElement('h1');
+    this.headBodyTitle.innerHTML = this.title;
+    this.headBody.appendChild(this.headBodyTitle);
+    this.head.appendChild(this.headBody);
+    this.pageContent.appendChild(this.head);
+  }
+
+  renderMenu() {
+    this.navElement1 = document.createElement('li');
+    this.navElement1.className = 'nav-element';
+    this.navElement1.innerHTML = 'Home';
+
+    this.navElement2 = document.createElement('li');
+    this.navElement2.className = 'nav-element';
+    this.navElement2.innerHTML = 'Menu';
+
+    this.navElement3 = document.createElement('li');
+    this.navElement3.className = 'nav-element';
+    this.navElement3.innerHTML = 'Contact';
+
+    this.nav = document.createElement('ul');
+    this.nav.className = 'nav';
+
+    this.nav.appendChild(this.navElement1);
+    this.nav.appendChild(this.navElement2);
+    this.nav.appendChild(this.navElement3);
+
+    this.tabs = document.createElement('div');
+    this.tabs.className = 'tabs';
+    this.tabs.appendChild(this.nav);
+    this.container.appendChild(this.tabs);
+  }
+
+  menu() {
+    this.navElement1.classList.add('active');
+    this.renderContent();
+
+    this.navElement1.addEventListener('click', () => {
+      this.activeMenu = 'Home';
+      this.navElement1.classList.add('active');
+      this.navElement2.classList.remove('active');
+      this.navElement3.classList.remove('active');
+      this.renderContent();
+    });
+
+    this.navElement2.addEventListener('click', () => {
+      this.activeMenu = 'Menu';
+      this.navElement1.classList.remove('active');
+      this.navElement2.classList.add('active');
+      this.navElement3.classList.remove('active');
+      this.renderContent();
+    });
+
+    this.navElement3.addEventListener('click', () => {
+      this.activeMenu = 'Contact';
+      this.navElement1.classList.remove('active');
+      this.navElement2.classList.remove('active');
+      this.navElement3.classList.add('active');
+      this.renderContent();
+    });
+  }
+
+  renderContent() {
+    if (this.activeMenu === 'Home') {
+      this.content.innerHTML = '';
+      this.content.appendChild(Home());
+    } else if (this.activeMenu === 'Menu') {
+      this.content.innerHTML = '';
+      this.content.appendChild(Menu());
+    } else {
+      this.content.innerHTML = '';
+      this.content.appendChild(Contact());
+    }
```

Comment:
You could simplify this a bit if you take advantage of the fact that your functions are using the same names as the `activeMenu`:

```js
renderContent() {
  const content = { Home, Menu, Contact };
  this.content.innerHTML = '';
  this.content.append(content[this.activeMenu]());
}
```